### DISCUSSION TITLE
Cancel to get old div without second sever request?

### DISCUSSION THREAD

**Discussion by rnkn** [bigskysoftware/htmx#2207] [Q&A]
Hello!

I'm still getting my head around HTMX so please forgive the beginner question; in the [Click to Edit example](https://htmx.org/examples/click-to-edit/) the `button` element in the initial content sends the GET request to the server, which responds with the `form` content.

So far so good.

From what I can tell, clicking the cancel `button` sends another GET request to the server, which responds again with the initial content? Or is it possible to revert the content without a second server request, or am I making a fundamental misunderstanding?

**Comment by mdesco-crgl** [comment]
This might help: https://htmx.org/extensions/restored/
